.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
@Ali-Tehrani, is this file meant to be part of the repo or is it your editor's config file?